ui/src/pages/help-center/HelpCenter.constants.ts

```diff
@@ -0,0 +1,110 @@
+export const cardIds = {
+  pmmDocs: 'pmm-docs',
+  support: 'support',
+  forum: 'forum',
+  pmmDump: 'pmm-dump',
+  pmmLogs: 'pmm-logs',
+  tips: 'tips',
+};
+
+export const startIcon = {
+  download: 'download',
+  map: 'map',
+};
+
+export const CardsData = [
+  {
```

Comment:
One thing you could add here is a boolean prop like `requiredRole` / `adminOnly` / `roles` (list of org roles), which I think would simplify the auth check.